src/App.js

```diff
@@ -25,14 +33,19 @@ function App() {
 
   return (
     <>
+      <Navbar />
       <h1>{message}</h1>
       <Routes>
+        <Route
+          path="/"
+          element={isLoggedIn ? <Home /> : <Navigate to="/signIn" />}
+        ></Route>
         <Route path="/register" element={<UserRegistration />}></Route>
-        <Route path="/signIn" element={<Contact />}></Route>
+        <Route path="/signIn" element={<SignIn />}></Route>
         <Route path="/home" element={<Home />}></Route>
         <Route path="/logout" element={<Logout />}></Route>
+        <Route path="/logout" element={<Logout />}></Route>
```

Comment:
I see you're rendering the Logout component in the navbar as well. Is this just here for testing purposes? Either way, I think the Logout component can be removed from App.js.

Reply:
I could be wrong, but I _think_ the way it's set up we still need it here (though right now it's duplicated so we can remove one of line 46 or 47) to define the route for `react-router-dom`. 

Maybe the routes should be declared elsewhere/earlier though to not make it confusing vs the actual content of the `App` component? I'm not 100% sure on best practices around that

Reply:
ohh nvm I see; the component for `<Logout>` is just a button not a full page; so there's nothing really to route to. I agree then with @nikkigraybeal  we can remove line 46 and 47 here